rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -2925,11 +2938,24 @@ public boolean isConnected(@HeaderParam("sessionid")
     public String login(@FormParam("username") String username, @FormParam("password") String password)
             throws LoginException, SchedulerRestException {
         try {
-            if ((username == null) || (password == null)) {
+
+            if ((username == null || username.trim().isEmpty()) || (password == null || password.trim().isEmpty())) {
```

Comment:
Same remark for the use of Apache Commons here.

Reply:
done